src/engine/client/client.cpp

```diff
@@ -591,7 +591,7 @@ int CClient::LoadData()
 
 // ---
 
-void *CClient::SnapGetItem(int SnapID, int Index, CSnapItem *pItem)
+void *CClient::SnapGetItem(int SnapID, int Index, CSnapItem *pItem) const
```

Comment:
Also can't be marked as `const`. Either change the return type to `const void *` or remove the `const`. Same applies to other functions below.